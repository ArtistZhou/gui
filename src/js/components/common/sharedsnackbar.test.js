import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import thunk from 'redux-thunk';
import configureStore from 'redux-mock-store';
import SharedSnackbar from './sharedsnackbar';

const mockStore = configureStore([thunk]);
const store = mockStore({
  app: {
    snackbar: {}
  }
});

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <SharedSnackbar snackbar={{ maxWidth: 200 }} />
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
