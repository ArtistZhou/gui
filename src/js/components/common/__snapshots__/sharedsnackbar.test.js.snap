// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SharedSnackbar Component renders correctly 1`] = `null`;
